org.srb2.SRB2.yaml

```diff
@@ -0,0 +1,60 @@
+app-id: org.srb2.SRB2
+runtime: org.freedesktop.Platform
+runtime-version: '19.08'
+sdk: org.freedesktop.Sdk
+command: srb2
+finish-args:
+  - --share=ipc
+  - --share=network
+  - --socket=fallback-x11
+  - --socket=pulseaudio
+  - --socket=wayland
+  - --device=all
+  - --persist=.srb2
+modules:
+  - shared-modules/glu/glu-9.json
+  - name: game-music-emu
+    sources:
+      - type: archive
+        url: https://bitbucket.org/mpyne/game-music-emu/downloads/game-music-emu-0.6.3.tar.xz
+        sha256: aba34e53ef0ec6a34b58b84e28bf8cfbccee6585cebca25333604c35db3e051d
+    buildsystem: cmake-ninja
+    cleanup:
+      - /include
+      - /lib/*.so
+      - /lib/pkgconfig
+  - name: libopenmpt
+    sources:
+      - type: archive
+        url: https://lib.openmpt.org/files/libopenmpt/src/libopenmpt-0.4.12+release.autotools.tar.gz
+        sha256: 0ccd64476e6c8a084277e7093c4034d702e7999eeffd31adc89b33685e725e60
+    config-opts:
+      - --disable-examples
+      - --disable-openmpt123
+      - --disable-static
+      - --disable-tests
+      - --without-portaudio
+      - --without-portaudiocpp
+    cleanup:
+      - /include
+      - /lib/*.la
+      - /lib/*.so
+      - /lib/pkgconfig
+      - /share/doc
+  - name: srb2
+    sources:
+      - type: git
+        url: https://github.com/STJr/SRB2
+        tag: SRB2_release_2.2.3
+        commit: 225095afa2fb1c61d12cf96c1b7c56cb4dbb4350
+      - type: file
+        path: org.srb2.SRB2.desktop
+      - type: file
+        path: org.srb2.SRB2.metainfo.xml
+    buildsystem: simple
+    build-commands:
+      - 'make -C src -j $FLATPAK_BUILDER_N_JOBS LINUX64=1 NOUPX=1 SDL=1'
```

Comment:
Doen't it support cmake?

Reply:
The Makefiles are the recommended way to build the upstream project. The CMake config in place does not allow builds without the proprietary assets, which is major shortcoming.